app/services/create_performance_json.service.js

```diff
@@ -6,9 +6,11 @@ const lodash = require('lodash')
 const filteredServices = require('./get_filtered_services.service')
 const performanceDataService = require('./get_performance_report.service')
 
+const DATE_FORMAT = 'yyyy-MM-DD'
+
 async function createPerformanceJson() {
-  const fromDate = moment.utc().subtract(7, 'days').startOf('day').format()
-  const toDate = moment().utc().startOf('day').format()
+  const fromDate = moment.utc().subtract(7, 'days').format(DATE_FORMAT)
```

Comment:
Shouldn't this be from September 2016 rather than 7 days ago?

Reply:
👍